### ISSUE TITLE
Question: How will Keen implementation of GPS affect Waypoints section of Essentials.

### ISSUE THREAD

**Issue by markam67** [Tyrsis/EssentialsPlugin#69]

**Comment by LordNex** [comment]
Was wondering the same thing myself.

**Comment by McyD** [comment]
I am wondering if keens implementation has now removed the hard-coded distance limit on the beacon/waypoints now.

**Comment by Tyrsis** [comment]
Keen's waypoints are not complete: allow no server control, do not allow faction waypoint management with permissions, do not allow colours, and are also weirdly tied to the useless chat they added.  Therefore I will not remove it, as most people actually prefer the old style of waypoints in some cases.

**Comment by markam67** [comment]
Yes until they provide server side waypoints that can be set by admin (at the very least) I am glad you are keeping it.
